botnim/config.py

```diff
@@ -14,10 +15,68 @@
 
 def get_logger(name: str) -> logging.Logger:
     """Get a logger instance for the given name"""
-    return logging.getLogger(name)
+    logger = logging.getLogger(name)
+    logger.setLevel(logging.INFO)
+    if not logger.handlers:
+        handler = logging.StreamHandler()
+        handler.setFormatter(logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s'))
+        logger.addHandler(handler)
+    return logger
 
 
 # Embedding model settings
 DEFAULT_EMBEDDING_MODEL = 'text-embedding-3-small'
 DEFAULT_EMBEDDING_SIZE = 1536
 DEFAULT_BATCH_SIZE = 50
+
+# Constants
+VALID_ENVIRONMENTS = ['production', 'staging']
+DEFAULT_ENVIRONMENT = 'staging'
+
+def validate_environment(environment: str) -> str:
```

Comment:
This is something that is enforced by the cli library - why do we need to re-check it everywhere?

Reply:
This is indeed redundant in CLI flows, but it provides validation for programmatic calls that bypass the CLI

Reply:
should I move it out of the CLI ...?

Reply:
@noamoss

Reply:
for example, the `get_available_indexes` and `get_index_fields` could benefit from this kind of validation. BTW, I am not familiar enough with the server implementation, but this kind of implementation might also support endpoint calls (which are not cli).

Reply:
`get_available_indexes` and `get_index_fields` are called via the cli though, where the input is already validated. There should be one place that's responsible for each concern (e.g. input validation). In our case, cli.py is where we do input validation. The rest of the code should assume that input is validated.
(And just another way to look at it, why not validate the bot name as well on every function? and that rtl is a bool? and query_text is a string?)

Reply:
ok, fixed.